packages/render/src/use-frame.tsx

```diff
@@ -344,6 +330,22 @@ export function useFrame<
             window.open(location, "_blank")?.focus();
           }
 
+          return;
+        } else if ("message" in responseData) {
+          const stackItem: FrameStackMessage = {
+            ...frameStackPendingItem,
+            responseStatus: response.status,
+            speed: computeDurationInSeconds(startTime, endTime),
+            status: "message",
+            message: responseData.message,
+          };
+
+          dispatch({
+            action: "DONE",
+            pendingItem: frameStackPendingItem,
+            item: stackItem,
+          });
+          window.alert(responseData.message);
```

Comment:
are we sure we want to do a window alert in the use frame package?

Reply:
good catch, meant to remove